skills/refactor/phases/overhaul-delegate.md

```
@@ -103,19 +103,24 @@ See `overhaul-review.md` for detailed criteria.
 
 ## State Updates
 
-```bash
+Use `mcp__workflow-state__workflow_set` with the featureId for state updates:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)